service/src/main/java/bio/terra/cbas/runsets/outputs/OutputGenerator.java

```diff
@@ -6,17 +6,21 @@
 import org.databiosphere.workspacedata.model.RecordAttributes;
 
 public class OutputGenerator {
+
   public static RecordAttributes buildOutputs(
-      List<WorkflowOutputDefinition> outputDefinitions, Object cromwellJsonObject) {
-    RecordAttributes outputParams = new RecordAttributes();
-    for (WorkflowOutputDefinition outputParam : outputDefinitions) {
-      String parameterName = outputParam.getOutputName();
-      Object outputValue;
-      outputValue = ((Map<String, Object>) cromwellJsonObject).get(parameterName);
-      String attributeName = outputParam.getRecordAttribute();
+      List<WorkflowOutputDefinition> outputDefinitions, Object cromwellOutputs) throws Exception {
```

Comment:
TOL/Nit: Can we be more specific than "Exception"?

Reply:
Yeah I can add a new Exception type.

Reply:
Added new Exception type called `WorkflowOutputNotFoundException`